flow/project.py

```diff
@@ -3614,15 +3614,19 @@ def _convert_jobs_to_aggregates(self, jobs):
         return aggregates
 
     @contextlib.contextmanager
-    def _potentially_buffered(self):
+    def _buffered(self):
         """Enable the use of buffered mode for certain functions."""
-        if self.config["flow"].as_bool("use_buffered_mode"):
-            logger.debug("Entering buffered mode...")
-            with signac.buffered():
-                yield
-            logger.debug("Exiting buffered mode.")
-        else:
+        if not self.config["flow"].as_bool("use_buffered_mode"):
+            warnings.warn(
+                "The use_buffered_mode config option is deprecated as of 0.12 "
+                "and will be removed in 0.14. Buffered mode is always enabled "
```

Comment:
If we keep the buffering mode always enabled then I think we should remove this in 0.13 instead.

Reply:
I read our deprecation policy again and that makes sense. https://docs.signac.io/projects/flow/en/latest/support.html#deprecation-policy
```suggestion
                "and will be removed in 0.13. Buffered mode is always enabled "
```